### DISCUSSION TITLE
Couple of Development Questions

### DISCUSSION THREAD

**Discussion by joshuabillington** [flyntwp/flynt#415] [Q&A]
How do I use the Grid Posts Archive Component? - I saw the question on here about it, but it didn't seem to work in my case. 

I also was experimenting with calendar plugins, and was wondering if someone had a recommendation as to the best one to use? And how it might be integrated? 

Thanks in advance.

**Comment by timohubois** [comment]
@joshuabillington have you found answers to your questions?
